build.gradle

```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-
+    ext.kotlin_version = '1.3.21'
```

Comment:
ist this kotlin version defined somewhere else on Dependencies file? Why are you explicitly putting it here?

Reply:
I'm not sure why I did this. It's been some time since I made these changes. I'll investigate and let you know soon.

Reply:
I used the automated option to update Kotlin, and it added this line here. I've fixed this now. Thanks for pointing this out!